fission-web-server/library/Fission/Web/Server/Auth/Error.hs

```diff
@@ -1,10 +1,10 @@
 module Fission.Web.Server.Auth.Error (Error (..)) where
 
-import qualified Servant.Server           as Server
+import           Servant.Server           hiding (BasicAuthResult (..))
```

Comment:
I assume BasicAuthResult had some conflict, but I don't see it? 🤔

Reply:
`BasicAuthResult` is the type, which contains a value `NoSuchUser`, which is the conflict. I guess we could change that to `NoUserFound` or something to disambiguate, but honestly 🤷‍♀️